examples/make_eeg_bids.py

```diff
@@ -170,7 +170,7 @@
                     output_path=output_path,
                     kind=kind,
                     event_id=trial_type,
-                    overwrite=False
+                    write_mode='overwrite'
                     )
```

Comment:
If overwrite is working now, you can screen the example and remove superfluous "sh.rmtree" commands such as this one:

https://github.com/mne-tools/mne-bids/blob/2da2a727ea125372b753fe720fd09f7db044796f/examples/make_eeg_bids.py#L143

Reply:
I will remove this one and add the `write_mode = 'clear'` to the subsequent call on `raw_to_bids`, but the other `sh.rmtree(~)` call would need to stay because `raw_to_bids` is being called multiple times in a loop...